src/pyconverter/xml2py/ast_tree.py

```diff
@@ -244,6 +167,23 @@ def children(self):
         """Children of the tree."""
         return self._content
 
+    @property
+    def children_types(self):
+        """Types of the children."""
+        return [type(child).__name__ for child in self.children]
+
+    def get_children_by_type(self, type_name):
+        """Return children of a specific type."""
```

Comment:
```suggestion
        """Get children of a specific type."""
```
Better to match the verb in the function name.